pkgs/top-level/python-packages.nix

```diff
@@ -7817,17 +7860,14 @@ let
       sed -i '0,/from numpy.distutils.core/s//import setuptools;from numpy.distutils.core/' setup.py
     '';
 
-    preBuild = ''
-      export BLAS=${pkgs.openblas} LAPACK=${pkgs.openblas}
-    '';
+    preBuild = self.numpyScipyPrebuild;
```

Comment:
Would it be possible to move the `numpyScipyPrebuild` code into this expression instead of defining it in the top-level package set?